axum-extra/Cargo.toml

```diff
@@ -32,6 +32,7 @@ protobuf = ["dep:prost"]
 query = ["dep:serde", "dep:serde_html_form"]
 spa = ["tower-http/fs"]
 typed-routing = ["dep:axum-macros", "dep:serde", "dep:percent-encoding"]
+with_rejection = []
```

Comment:
I don't think this needs to be behind a feature flag, since it doesn't pull in any additional dependencies.